participants/GammaCloud.md

```diff
@@ -0,0 +1,100 @@
+# GammaCloud
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Our Project Name is GammaCloud. Our Team Slack Handles is `UK64776JJ`, and with Display name `RockYang`.
+
+## (1) Brief Project Description
+
+GammaCloud is a cloud file system baseed on Filecoin and IPFS.
+
+Any enterprise users and individual users can back up their data to GammaCloud. Usually users can only see the data uploaded by themselves. 
+
+We will also upload some open datasets to allow user to download publicly.
+
+## (2) Link to Project UI
+
+1. [GammaCloud Demo Preview](https://gfs.xjxh.pro)
+2. [GammaCloud repository](https://github.com/xjxh/gamma-cloud)
+
+## (3) What does your application/UI do?
+
+1. If users want to upload theirs file, they must register first.
+2. User login the system to upload and download theirs files. For public data sets, anyone can access without login.
+3. For enterprise users, we also provider APIs for push and pull files.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+We do not use any curated dataset. Our data comes from 2 chanels:
+
+1. Users upload theirs data to our file system.
+2. Some Software mirrors like: linux kernel, linux release iso file, opensource database mirrors, k8s eg...
```

Comment:
Please ensure you have data rights for this.

Reply:
These are public files, we can download them from their official website or from public mirror repositories such as AliYun's mirror repository.